### DISCUSSION TITLE
Archery更改docker-compose.yml中的mysql密码后容器Archery无法连接mysql

### DISCUSSION THREAD

**Discussion by huazi-zhao** [hhyo/Archery#2463] [Q&A]
当我把docker-compose.yml中的mysql密码更改后拉取的“Archery容器”初始化时连接不上“mysql容器”是为什么
MySQLdb._exceptions.OperationalError: (1045, "Access denied for user 'root'@'172.19.0.2' (using password: YES)")

**Comment by finovy** [comment]
请修改.env文件中的 DATABASE_URL 配置
